ansible/roles/ocp-workload-cnad-app-modernization/files/crw-cluster.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: org.eclipse.che/v1
```

Comment:
This should not be necessary. There already is a workload `ocp4_workload_crw` that deploys CRW. No need to re-invent the wheel. the CheCluster `spec` is a parameter for that workload.